test/controllers/downloads_controller_test.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'test_case'
+
+class DownloadsControllerTest < ActionController::TestCase
+  test 'download' do
+    user = create(:public_health_user)
+    download = create(:download, user_id: user.id)
+    text = StringIO.new('text')
+    download.exports.attach(io: text, filename: 'text.txt', content_type: 'application/text')
+    sign_in user
+    get :download, params: { id: download.id }
+    assert_response :success
```

Comment:
would it be possible to verify the content of the download as well

Reply:
That seems duplicative of the system test.

Reply:
that's fair